### PR TITLE
Upload cloud-init logs

### PR DESCRIPTION
**Pull request by jlausuch**
VR: https://openqa.suse.de/tests/13620400

### CODE REVIEW

File: tests/boot/cloud_init.pm [hunk 1]
```diff
@@ -10,9 +10,11 @@ use Mojo::Base 'opensusebasetest';
 use testapi;
 use serial_terminal 'select_serial_terminal';
 
-sub upload_journal {
+sub post_boot_logs {
     script_run("journalctl --no-pager -o short-precise > /tmp/journal_after_boot.log");
-    upload_logs "/tmp/journal_after_boot.log";
+    upload_logs("/tmp/journal_after_boot.log");
```
Comment (mloviska) [inline tests/boot/cloud_init.pm:15]: You can merge these lines using lib/Utils/Logging.pm::save_and_upload_log()

File: tests/boot/cloud_init.pm [hunk 2]
```diff
@@ -26,11 +28,11 @@ sub run {
     record_info('JOURNAL', script_output('journalctl --no-pager -u cloud-init'));
     validate_script_output('cat /tmp/cloud-message', sub { m/^cloud-init was here$/ });
 
-    upload_journal;
+    post_boot_logs;
```
Comment (mloviska) [inline tests/boot/cloud_init.pm:31]: this should go to `post_run_hook`

File: tests/boot/cloud_init.pm (hunk 1 above)
Comment (jlausuch) [reply tests/boot/cloud_init.pm]: done

File: tests/boot/cloud_init.pm (hunk 2 above)
Comment (jlausuch) [reply tests/boot/cloud_init.pm:31]: done

**Review by foursixnine — APPROVED** [review]

**Pull request merged**